src/gui/widgets/LcdWidget.cpp

```diff
@@ -177,21 +185,24 @@ void LcdWidget::paintEvent( QPaintEvent* )
 	}
 
 	// Right Margin
-	p.drawPixmap( QRect( 0, 0, m_marginWidth-1, m_cellHeight ), *m_lcdPixmap, 
-			QRect( charsPerPixmap*m_cellWidth, isEnabled()?0:m_cellHeight,
-				m_cellWidth / 2, m_cellHeight ) );
+	p.drawPixmap(QRect(0, 0, m_seamlessRight ? m_marginWidth : m_marginWidth - 1, m_cellHeight),
```

Comment:
Just a question, why the `m_seamlessRight` draws a rect with width of `m_marginWidth` and the one that isn't seamless draws a rect with width `m_marginWidth - 1`?

I was expecting it to be width `0` for seamless and `m_marginWidth - 1` for not seamless.

Reply:
Frankly I have no idea. I probably tweaked it empirically back at the time, i.e. tried different values until it worked correctly. I can try to build the original Microtuner PR and try different values to see what happens if you want.

Reply:
Cool, could you try using 0 for the seamless? I think that would make sense. Having `m_marginWidth` is probably not causing any issues now because nothing is being drawn on this extra portion of the pixmap? I'm not sure tbh

Reply:
I just tested both versions and as far as I can tell they look exactly the same, so replaced it with zero as you suggested.